lib/flex_commerce_api/api_base.rb

```diff
@@ -13,18 +13,21 @@
 require "flex_commerce_api/json_api_client_extension/has_many_association_proxy"
 require "flex_commerce_api/json_api_client_extension/builder"
 require "flex_commerce_api/json_api_client_extension/flexible_connection"
-require "flex_commerce_api/json_api_client_extension/resource"
+require "flex_commerce_api/json_api_client_extension/parsers/parser"
+
 module FlexCommerceApi
   #
   # Base class for all flex commerce models
   #
-  class ApiBase < ::FlexCommerceApi::JsonApiClientExtension::Resource
+  class ApiBase < JsonApiClient::Resource
```

Comment:
Tried using `:FlexCommerceApi::JsonApiClientExtension::Resource` but ran into all sorts of loading issues.

This was the only class that worked :(